rm-datasource/src/main/java/io/seata/rm/datasource/sql/druid/oracle/OracleDeleteRecognizer.java

```diff
@@ -83,7 +85,15 @@ public String getWhereCondition(final ParametersHolder parametersHolder, final A
         }
         StringBuilder sb = new StringBuilder();
         OracleOutputVisitor visitor = super.createOracleOutputVisitor(parametersHolder, paramAppenderList, sb);
-        visitor.visit((SQLBinaryOpExpr) where);
+        if (where instanceof SQLBinaryOpExpr) {
```

Comment:
how about move to a common place such as   BaseRecognizer?

Reply:
maybe yes, but have to separate like getMysqlWhereCondition(expr) and getOracleWhereCondition(expr) due to different output visitor.

Reply:
fixed. I will change the unit test later